packages/core/mesh/network-manager/src/transport/libdatachannel-transport.test.ts

```diff
@@ -0,0 +1,96 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+//
+// Copyright 2020 DXOS.org
+//
+
+import { Duplex } from 'stream';
+
+import { sleep, TestStream } from '@dxos/async';
+import { log } from '@dxos/log';
+import { afterTest, describe, test } from '@dxos/test';
+
+import { LibDataChannelTransport } from './libdatachannel-transport';
+import { SimplePeerTransport } from './simplepeer-transport';
+
+describe.only('LibDataChannelTransport', () => {
```

Comment:
.only causes the rest of the tests in this package to be skipped - remove before merging